app/src/main/java/io/github/droidkaigi/confsched2018/presentation/NavigationController.kt

```diff
@@ -163,12 +163,22 @@ class NavigationController @Inject constructor(private val activity: AppCompatAc
             }
             uri
         }
+
+        val packageName = CustomTabsHelper.getPackageNameToUse(activity)
```

Comment:
Sorry, If you can please rename to chromePackageName 🙏

Reply:
OK, that is better
https://github.com/DroidKaigi/conference-app-2018/pull/557/commits/c9d5ee02f6159f48b0886aaf1a1e33ea43a2d086